packages/connect-thegraph/src/core/GraphQLWrapper.ts

```diff
@@ -7,11 +7,6 @@ import { DocumentNode } from 'graphql'
 import { ParseFunction, QueryResult } from '../types'
 import { pipe, subscribe } from 'wonka'
 
```

Comment:
I removed these types so that they can be handled in a more generic way, without requiring users to find obscure types. They're very intuitive imo. We can add them back if you don't like it.